### ISSUE TITLE
listOsTypes respond returns isuserdefined as string

### ISSUE THREAD

**Issue by resmo** [apache/cloudstack#2529]
##### ISSUE TYPE

 * Bug Report
 * Enhancement

##### COMPONENT NAME

~~~
API
~~~

##### CLOUDSTACK VERSION

~~~
4.11
~~~

##### CONFIGURATION

N/A 

##### OS / ENVIRONMENT

N/A 

##### SUMMARY

listOsTypes returns `isuserdefined` as json string, instead of json bool

##### STEPS TO REPRODUCE

~~~
 $ cs listOsTypes id=1ee61d39-4b9a-11e7-8fea-0242ac110003 --region simulator                                          
~~~

##### EXPECTED RESULTS

~~~json
{
  "count": 1, 
  "ostype": [
    {
      "description": "Windows XP SP3 (32-bit)", 
      "id": "1ee61d39-4b9a-11e7-8fea-0242ac110003", 
      "isuserdefined": false, 
      "oscategoryid": "1ede7e9a-4b9a-11e7-8fea-0242ac110003"
    }
  ]
}
~~~

##### ACTUAL RESULTS

~~~json
{
  "count": 1, 
  "ostype": [
    {
      "description": "Windows XP SP3 (32-bit)", 
      "id": "1ee61d39-4b9a-11e7-8fea-0242ac110003", 
      "isuserdefined": "false", 
      "oscategoryid": "1ede7e9a-4b9a-11e7-8fea-0242ac110003"
    }
  ]
}
~~~

**Comment by rhtyd** [comment]
Closing as Fixed in PR #2632

**Comment by resmo** [comment]
no need to close it manually, github will close it after PR is merged.
